src/store/annotator/annotatorListeners.ts

```diff
@@ -23,7 +24,10 @@ startAppListening({
     const { imageViewer, data, annotator } = listenerAPI.getState();
     const { annotationState, annotationTool } = action.payload;
 
-    if (!annotationTool || annotationState !== AnnotationStateType.Annotated)
+    if (
+      annotationTool instanceof BlankAnnotationTool ||
```

Comment:
This guard isn't strictly necessary, since nothing "bad" would happen if we use the below logic on `BlankAnnotationTool`, but it's a fair bit of needless computation so just return early.